packages/bundler-plugin-core/src/utils.ts

```diff
@@ -165,3 +166,23 @@ function lookupPackageJson(cwd: string, stopAt: string): PackageJson | undefined
   const newCwd = path.dirname(path.resolve(jsonPath + "/.."));
   return lookupPackageJson(newCwd, stopAt);
 }
+
+/**
+ * Deterministically hashes a string and turns the hash into a uuid.
+ */
+export function stringToUUID(str: string): string {
```

Comment:
l: Can we have a test that enforces this will return a uuid - sanity check mostly that it behaves ok across Node versions.

Reply:
Added a small test: 4154b4a974e30183befed53d36674a4e80754147